### PR TITLE
[mlir] make mlir related stuff build on macos

### PR DESCRIPTION
**Pull request by freedomtan**
`bazel build --config opt tensorflow/compiler/mlir/...` doesn't work on macOS.
1. `exp10()` is not a standard C function, macos has `__exp10()`
2. include `"mlir/StandardOps/Ops.h"` earlier to avoid `TRUE` and`FALSE` being defined in `<mach/boolean.h>`

### CODE REVIEW

File: tensorflow/compiler/mlir/lite/ir/tfl_traits.h [hunk 1]
```diff
@@ -22,6 +22,11 @@ limitations under the License.
 #include "mlir/Support/LLVM.h"  // TF:local_config_mlir
 #include "tensorflow/compiler/mlir/lite/utils/quantization_utils.h"
 
+// exp10() is not a standard function
```
Comment (jpienaar) [inline tensorflow/compiler/mlir/lite/ir/tfl_traits.h]: Could you update the call site to use pow instead?

**Comment by jpienaar** [comment]
One suggestion, else it looks good, thanks

File: tensorflow/compiler/mlir/lite/ir/tfl_traits.h (hunk 1 above)
Comment (freedomtan) [reply tensorflow/compiler/mlir/lite/ir/tfl_traits.h]: sure.

**Comment by freedomtan** [comment]
rebased to resolve conflicts and replaced `exp10(X)` with `pow(10.0, X)`

**Review by jpienaar — DISMISSED** [review]
Looks good, I've also updated TRUE & FALSE to avoid needing to make the non-pow changes.

**Comment by freedomtan** [comment]
Thanks for your work and review. BTW, I cannot build related stuff after 22385cc65e2, either reverting it or more modifications needed.

**Comment by jpienaar** [comment]
Could you rebase? We've changed function to be value typed. If that doesn't work, I'll apply your change to head and commit it.

**Comment by freedomtan** [comment]
Yes, 604988b works. The only change needed is replacing `exp10()`. I'll update with additional `"mlir/StandardOps/Ops.h"`removed.

**Review by jpienaar — APPROVED** [review]

**Pull request merged**